docs/resources/device.md

```diff
@@ -0,0 +1,52 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "fmc_device Resource - terraform-provider-fmc"
+subcategory: "Devices"
+description: |-
+  This resource can manage a Device.
+---
+
+# fmc_device (Resource)
+
+This resource can manage a Device.
+
+## Example Usage
+
+```terraform
+resource "fmc_device" "example" {
+  name             = "device1"
+  host_name        = "10.0.0.1"
+  license_caps     = ["BASE"]
+  reg_key          = "key1"
+  access_policy_id = "fmc_access_control_policy.test.id"
```

Comment:
How to do this better @danischm ? Generally, I think we approach `example` in a deficient way. Choices:

1. Current choice: `"00000000-0000-0000-0000-000000000000"` Users guess where to get the UUID from, even though it's usually the harder part of understanding how any resource fits in.
2. My suggestion: `fmc_access_control_policy.test.id`. Eliminate guessing, but the example itself is not executable.
3. More work: `fmc_access_control_policy.test.id` and also include a minimal `resource fmc_access_control_policy { name = ... }`. No guessing and readily executable (e.g. [aws_instance](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance)).

Reply:
I mean `fmc_access_control_policy.test.id` without quotes, which seems impossible as-is.

Reply:
I would suggest to go with (1) for now and maybe add a comment/example in the attribute description on where to get the access policy id from.

Reply:
done, thanks